functions/output_display.py

```diff
@@ -0,0 +1,27 @@
+import functions.trigonometry as trig
+from decimal import *
+import math
+
+def rad(deg: float) -> float:
+    """Returns the radians corresponding to input angle
+       Args:
+           deg (float): Angle in degree
+       Returns:
+           float: Radian of input
+    """
+
+    return float(Decimal(deg) * Decimal(trig.generate_pi()) / Decimal(180.0))
+     # TO SHOW ACCURACY: return float(Decimal(deg) * Decimal(math.pi) / Decimal(180.0))
```

Comment:
I've kept these in the file, commented out to be able to easily switch back and forth with math.pi and our variation of it. I see that because our version is not as precise, it causes this function to not be as precise. 

By testing it with this commented out function, I can test my function without relying on the precision of the underlying functions. 

I'd like to keep this here in the PR until we fix precision issues. Will remove before submitting.